Work on the first 5 G's (Glossary) (#160)
canonical/ubuntu-server-documentation: reference/glossary.rst

    FW
        *Work in Progress*

    gcplogs

>         A driver that allow logs to be forwarded from a Docker container running in Google Cloud to the Google Cloud Logging service.

Author: ```suggestion
        A storage driver that allows logs to be forwarded from a Docker container running in Google Cloud to the Google Cloud Logging service.
```

>         A storage driver that allows logs to be forwarded from a Docker container running in Google Cloud to the Google Cloud Logging service.

Author: ```suggestion
        A logging driver that allows logs to be forwarded from a Docker container running in Google Cloud to the Google Cloud Logging service.
```

Author: See https://docs.docker.com/engine/logging/drivers/gcplogs/

>         Related topic(s): Containers

Author: ```suggestion
        Related topic(s): Containers, and Observability.
```
